pyproject.toml

```diff
@@ -1,10 +1,10 @@
 [build-system]
 build-backend = "setuptools.build_meta"
 requires = [
-    "cython>=0.29",
+    "cython>=0.29.33",
     "oldest-supported-numpy",
-    "setuptools>=45.0",
-    "wheel>=0.37.0",
+    "setuptools>=67.1.0",
+    "wheel>=0.38.4",
 ]
```

Comment:
Interesting... why should these minimum requirements be increased? Are there breaking changes?